specs-go/config.go

```diff
@@ -495,7 +495,7 @@ type WindowsNetwork struct {
 // WindowsHyperV contains information for configuring a container to run with Hyper-V isolation.
 type WindowsHyperV struct {
 	// UtilityVMPath is an optional path to the image used for the Utility VM.
-	UtilityVMPath string `json:"utilityvmpath,omitempty"`
+	UtilityVMPath string `json:"utilityVMPath,omitempty"`
```

Comment:
On the unabbreviation front, this could be `utilityVirtualMachinePath`.  I don't see a way to draw a hard line for this, and it's orthogonal to camelCasing, but I thought I'd point out that `def` → `default` style expansion is something of a slippery slope.  My own fuzzy possition is that I like `def` → `default` but am less enthusiastic about `VM` → `VirtualMachine`.  And while I think `blkioThrottleWriteIOPSDevice` is a bit hard to read, it may be a better choice than `throttleWriteOperationsPerSecondPerDevice`.  But if anyone does see a way to draw a good line, it would be nice to document that line in `style.md`.